src/test/java/com/google/sps/TestCI.java

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class TestCI {
+
+  @Test
+  public void testPass1() {
+    assertEquals("Expected", "Expected");
+  }
+
+  @Test
+  public void testPass2() {
+    assertEquals(3, 1 + 2);
+  }
+
+  /* @Test
+  public void testFail1() {
+    assertEquals(2, 3);
+  } */
```

Comment:
Should generally avoid committing commented out code :). 

This could have been added in a separate commit

Reply:
My bad - I forgot :(  I left it there to easily test out the failing status of CI here on GitHub. 

And I apologize for merging too early -- I got a little too excited about the first intern-reviewed PR! It won't happen again :)